docs/source/api/neuralhydrology.datasetzoo.datasetregistry.rst

```diff
@@ -0,0 +1,7 @@
+DatasetRegistry
+=====
```

Comment:
```suggestion
DatasetRegistry
===============
```

In rst files, the line with the symbol below the head has to have the exact same lengths as the heading. This should also fix the doc CI test that fails.